examples/networkconnectivity/v1beta1/serviceconnectionpolicy.yaml

```
@@ -0,0 +1,52 @@
+apiVersion: networkconnectivity.gcp.upbound.io/v1beta1
+kind: ServiceConnectionPolicy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: networkconnectivity/v1beta1/serviceconnectionpolicy
+  labels:
+    testing.upbound.io/example-name: default
+  name: default
+spec:
+  forProvider:
+    description: my basic service connection policy
+    location: us-central1
+    networkSelector:
+      matchLabels:
+        testing.upbound.io/example-name: producer_net
+    pscConfig:
+    - limit: 2
```

We get the following error:
```
case.go:363: ServiceConnectionPolicy.networkconnectivity.gcp.upbound.io "default" is invalid: [spec.forProvider.pscConfig: Invalid value: "array": spec.forProvider.pscConfig in body must be of type object: "array", <nil>: Invalid value: "null": some validation rules were not checked because the object was invalid; correct the existing errors to complete validation]
```

Since the `pscConfig` field is an object, can you try the following?

```suggestion
    pscConfig:
      limit: 2
```

---

That is correct, also for some reason `limit` is a string, see https://networkconnectivity.googleapis.com/$discovery/rest?version=v1 
```
    pscConfig:
      limit: "2"
```